api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -99,6 +99,11 @@ message RateLimit {
   // Disabled by default.
   XRateLimitHeadersRFCVersion enable_x_ratelimit_headers = 8
       [(validate.rules).enum = {defined_only: true}];
+
+  // Disables emitting the :ref:`x-envoy-ratelimited<config_http_filters_router_filter_x-envoy-ratelimited>` header
+  // in case of 429 responses.
+  // Defaults to false.
```

Comment:
No need to say this, it's implied by the proto3 defaults.

Reply:
I am happy to remove it. Although I copied it from failure_mode_deny so for the sake of consistency I would remove it also from there.